src/panels/ShareModelPanel/ShareModel.tsx

```diff
@@ -28,8 +95,26 @@ export default function ShareModel() {
     access: string;
   };
 
+  const userAlreadyHasAccess = (userName: string) => {
+    return users.some((userEntry: User) => userEntry.user === userName);
+  };
+
+  const handleRemoveUser = async (userName: string) => {
```

Comment:
Can you add the arguments to this as well. That way the function can be called without having to worry about the context that the function was declared in vs when it was called.

Reply:
If and when this function call moves out of scope, this change can be made.